developer-hub/src/reference_docs/glossary.rs

```diff
@@ -36,41 +36,71 @@
 //!
 //! #### Offchain
 //!
+//! Refers to operations conducted outside the blockchain's consensus mechanism. They are essential
+//! for enhancing scalability and efficiency, enabling activities like data fetching and computation
+//! without bloating the blockchain.
+//!
 //! #### Host Function:
 //!
+//! Host functions are functions provided by the runtime environment (the [host](#host)) to the Wasm
+//! runtime. These functions allow the Wasm code to interact with and perform operations on the
+//! [node](#node), like accessing the blockchain state.
```

Comment:
Same as above (or link to full node maybe, even though this is more referencing the 'client' impl)

```suggestion
//! node, like accessing the blockchain state.
```

Reply:
The definition is on line [21](https://github.com/paritytech/polkadot-sdk/pull/2273/files#diff-017e90ea029873de3b8919cc06cbfe59bb9875a21094c0ab00698171ce82cc12L21)